### PR TITLE
Add some red icons and fix some code formats

### PR DESCRIPTION
**Pull request by montlikadani**
I've given some code formatting and better looks to make it more spectacular and eye-catching.

Screenshots about changes:
kép
kép
kép

### CODE REVIEW

**Comment by acoop133** [comment]
What are you using to format the file?
I havent found one in vscode that outputs nice formatting.

**Comment by montlikadani** [comment]
I'm just using the web-editor https://github.com/openstyles/stylus, which have an option to beautify the code.

**Comment by acoop133** [comment]
Im going to have to find another way to format the file, possibly through Github Actions. Going through the editor seems like it will be a bit tedious over time.

Also on the side note, I need to figure out why GenerateStylusTestFile action failed to push for foreign PR's. 


But yeah i like your changes and when i figure out the solution to the formatting ill merge it. (Probably end of this week)

Cheers

**Comment by acoop133** [comment]
Alright, it looks like [Prettier extension](https://prettier.io/) in vscode works really well

**Comment by acoop133** [comment]
Just inspected the css changes, its very hard to track down whats changed. since the whole file is lit up because of the formatting. can you split the formatting into another commit?

File: Theme.css [hunk 1]
```diff
@@ -2428,7 +2428,7 @@ div#network.network > div > canvas {
 /* from https://github.com/Dunedan/mbp-2016-linux on Windows*/
 /* <g-emoji alias="heavy_check_mark"
     fallback-src="https://assets-cdn.github.com/images/icons/emoji/unicode/2714.png"
-    ios-version="6.0"></g-emoji> */
+    ios-version="6.0">✔️</g-emoji> */
```
Comment (acoop133) [inline Theme.css]: Can you provide a before and after screen shot of this?

**Comment by acoop133** [comment]
Feel free to hard reset everything i did on your branch.

**Comment by acoop133** [comment]
Ive loaded the changes into my browser and here are my thoughts.

### I will put my closing remarks here since this comment size is super long!

Let's drop everything except the timeline background color and formatting fix.

Im also curious to see what difference adding the tick did since im not using ios :P.

## What i like:
* Red timeline icon background for closing issue / PR. This clearly highlights at which point it is closed

## What im not really digging:

### Before:
image
image

### After:
* We're adding "eye catching" colour to other elements that dont need to draw user's attention(unsubscribe, lock, delete/bin), it feels quite out of place for this theme. However i do understand your thoughts behind them.
* I dont think re-oppening a Issue/PR needs critical red.
* Also bug: you can see that the disabled reopen pull request button is now lit up when its not interactable.
* preview lost its white colour

Theming github is very a finnicky process since their CSS usage is abysmal. so we have to be careful what we change as it may lead to unwanted changes.


image

image

image

image

**Comment by acoop133** [comment]
If you really like the red icons, you could create your own icon theme with same regex :P

File: Theme.css (hunk 1 above)
Comment (montlikadani) [reply Theme.css]: This is just a web editor issue that doesn't support special characters because I applied changes from the web editor here.

**Comment by montlikadani** [comment]
>     * preview lost its white colour
>     * The disabled reopen pull request button is now lit up when its not interactable.
>     * I dont think re-oppening a Issue/PR needs critical red.
Will be fixed

+ Added black background for issue filters, sorts etc.
kép
kép

**Comment by montlikadani** [comment]
+ Added color for commit links
kép

**Comment by acoop133** [comment]
@montlikadani I appreciate the effort, but as mentioned in my comment above, can you please separate the style changes and formatting into their own commits.

Additionally, separate each style change into its own commit with a precise title. It serves as a nice git history.

also mentioned above
```
Let's drop everything except the timeline background color and formatting fix.
Add a green background to re-oppening icon on the timeline. 
```
This also includes removing the icon colour changes.

I understand that I've been a pain in this PR, its just super hard going through your changes and verifying them because everything is committed together! 

Im going to take your points as most of it is really good! And form the changes to the standards of the guideline and co-commit with your username. so you're still marked as a contributor.
Im going to push these changes to the next release. id like to release v1.20.0 today.


P.S
Ive added a contribution section to the readme recently. My bad for not having this earlier, but your PR has inspired me to formulate one!

https://github.com/DarkThemeHub/GithubDarkTheme/blob/master/README.md#keep-it-simple-and-minimal

**Pull request closed without merging**